Docs/source/pages/advanced-usage.md

```diff
@@ -628,8 +715,13 @@ Logging inside the instrumented app can be configured from the driver using the
 
     .. code-tab:: py
 
-        altDriver.set_server_logging(AltLogger.File, AltLogLevel.Off);
-        altDriver.set_server_logging(AltLogger.Unity, AltLogLevel.Info);
+        altDriver.set_server_logging(AltLogger.File, AltLogLevel.Off)
+        altDriver.set_server_logging(AltLogger.Unity, AltLogLevel.Info)
+
+    .. code-tab:: robot
+
+        Set Server Logging    File    Off
+        Set Server Logging    Unity    Info
 
```

Comment:
Can you please put "Off" and "Info" params one below the other (same as File and Unity)? :D